src/lib/services/identity.ts

```diff
@@ -12,7 +12,7 @@ const getMemberByAlias = async (alias: string) => {
   }
 
   if (res.status === 404) {
-    throw new NotFound(`Member "${alias}" does not exist`)
+    throw new NotFound('capacity')
```

Comment:
it's the identity that hasn't been found, not the capacity

Reply:
it has been updated, thanks for a flag up, find and replace not always the best friend